pkg/services/object/put/distributed.go

```diff
@@ -33,7 +33,8 @@ type distributedTarget struct {
 	objMeta            object.ContentMeta
 	networkMagicNumber uint32
 
-	cnrClient *chaintcontainer.Client
+	cnrClient               *chaintcontainer.Client
+	metainfoConsistencyAttr string
```

Comment:
now it's handled as boolean, will behavior ever depend on the particular value?

Reply:
yes, in the latter commit. hm, i see, lets squash some commits